ddht/v5_1/rpc_handlers.py

```diff
@@ -196,6 +196,27 @@ async def do_call(self, params: NodeID) -> None:
         return None
 
 
+class LookupENRHandler(RPCHandler[Tuple[NodeID, Optional[int]], GetENRResponse]):
+    def __init__(self, network: NetworkAPI) -> None:
+        self._network = network
+
+    def extract_params(self, request: RPCRequest) -> Tuple[NodeID, Optional[int]]:
+        raw_params = extract_params(request)
+        raw_destination = raw_params[0]
```

Comment:
This can raise an `IndexError` since we haven't already validated the number of parameters (input validation is tedious...)